File: order-ms/src/main/java/com/nexeum/orderms/dto/response/Order.java
package com.nexeum.orderms.dto.response;

import lombok.Data;

import java.math.BigDecimal;
import java.util.Date;
import java.util.List;

@Data
public class Order {
    private Long orderId;
    private BigDecimal totalOrderPrice;
    private String orderStatus;
    List<ProductItem> orderItems;
    private Date createdDate;
    private Date updatedDate;
}

File: order-ms/dto/response/Order.cs
using System.ComponentModel.DataAnnotations;

namespace order_ms.dto.response
{
    public class Order
    {
        [StringLength(30)] 
        public string OrderId { get; set; } = string.Empty;
        public decimal TotalOrderPrice { get; set; }
        public string? OrderStatus { get; set; } = string.Empty;
        public List<ProductItem>? OrderItems { get; set; } = new List<ProductItem>();
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}